02-Appetisers/Sea_Ice_Coordinates.ipynb

```diff
@@ -55,2294 +55,8 @@
    "outputs": [
```

Comment:
<p>capitalise CICE4 and CICE5?</p><p>also perhaps just rephrase " to speed up opening data from CICE, or ..." ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/643/discussion/#comment-2508507779'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>